test/everypolitician/popolo/json_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class Everypolitician::Popolo::JsonTest < Minitest::Test
+  def test_legislative_periods
+    popolo_json = Everypolitician::Popolo::JSON.new(
+      events: [
+        { name: 'Election 1', classification: 'general election', start_date: '2014-01-01' },
+        { name: 'Term 2', classification: 'legislative period', start_date: '2015-01-01' },
+        { name: 'Term 1', classification: 'legislative period', start_date: '2010-01-01' },
+      ],
+    )
+    assert_equal 2, popolo_json.legislative_periods.size
+    assert_equal 2, popolo_json.terms.size
+    assert_equal 'Term 2', popolo_json.current_legislative_period.name
+    assert_equal 'Term 2', popolo_json.current_term.name
```

Comment:
Ah yep, good point. Fixed in https://github.com/everypolitician/everypolitician-popolo/pull/38/commits/26861e9ddef3d22a60da9fd163b081fb86706d44